crawler.js

```diff
@@ -394,10 +399,26 @@ export class Crawler {
 
     if (this.params.behaviorOpts) {
       await page.exposeFunction(BEHAVIOR_LOG_FUNC, (logdata) => this._behaviorLog(logdata, page.url(), workerid));
-      await this.browser.addInitScript(page, behaviors + `;\nself.__bx_behaviors.init(${this.params.behaviorOpts});`);
+      await this.browser.addInitScript(page, behaviors);
+      let customBehaviors = "null";
+
+      if (this.params.customBehaviors) {
+        customBehaviors = this.loadCustomBehaviors(this.params.customBehaviors);
+      }
+      await this.browser.addInitScript(page, `self.__bx_behaviors.init(${this.params.behaviorOpts}, false, ${customBehaviors});`);
     }
   }
 
+  loadCustomBehaviors(filename) {
+    let funcs = [];
+
+    for (const source of collectAllFileSources(filename, ".js")) {
+      funcs.push(source);
+    }
+
+    return "[" + funcs.join(",") + "]";
```

Comment:
We may want to test with multiple custom behaviors to make sure this works as expected

Reply:
I think it only works with the first script found in the directory.

Reply:
Updated to browsertrix-behaviors 0.5.1 and also added test for loading multiple behaviors at once, should now be working.

Reply:
(Testing two behaviors that activate on two different sites)